shell/pages/c/_cluster/explorer/index.vue

```diff
@@ -393,30 +444,26 @@ export default {
       this.canViewAgents = !!this.$store.getters['cluster/schemaFor'](WORKLOAD_TYPES.DEPLOYMENT);
 
       if (this.canViewAgents) {
-        if (!this.currentCluster.isLocal) {
-          try {
-            this.cattle = await this.$store.dispatch('cluster/find', {
-              type: WORKLOAD_TYPES.DEPLOYMENT,
-              id:   'cattle-system/cattle-cluster-agent'
-            });
-          } catch (err) {
-            this.cattle = null;
-          }
+        if (this.currentCluster.isLocal) {
+          await this.setAgentResource('fleetDeployment', WORKLOAD_TYPES.DEPLOYMENT, 'cattle-fleet-system/fleet-controller');
```

Comment:
can both these two be called at the same time, or can we not `await` them at all? same goes for the two requests below